client/blocks/eligibility-warnings/test/index.tsx

```diff
@@ -0,0 +1,181 @@
+			state
+		);
+
+		const notice = container.querySelector( '.notice.is-error' );
+
+		expect( notice ).toBeVisible();
+		expect( notice ).toHaveTextContent( /This site is not currently eligible/ );
+	} );
+
+	it( 'only renders a single notice when multible hard blocking holds exist', () => {
+		const state = createState( {
+			holds: [ 'BLOCKED_ATOMIC_TRANSFER', 'SITE_GRAYLISTED' ],
+		} );
+
+		const { container } = renderWithStore(
+			<EligibilityWarnings backUrl="" onProceed={ noop } />,
+			state
+		);
+
+		expect( container.querySelectorAll( '.notice' ) ).toHaveLength( 1 );
+	} );
+
+	it( 'dimly renders the hold card when AT has been blocked by a sticker', () => {
+		const state = createState( {
+			holds: [ 'BLOCKED_ATOMIC_TRANSFER', 'SITE_PRIVATE' ],
+		} );
+
+		const { getByTestId, getByText } = renderWithStore(
+			<EligibilityWarnings backUrl="" onProceed={ noop } />,
+			state
+		);
+
+		expect( getByTestId( 'HoldList-Card' ) ).toHaveClass( 'eligibility-warnings__hold-list-dim' );
+		expect( getByText( 'Help' ) ).toHaveAttribute( 'disabled' );
+		expect( getByText( 'Continue' ) ).toBeDisabled();
+	} );
+
+	it( 'renders warning notices when the API returns warnings', () => {
+		const state = createState( {
+			warnings: [
+				{ name: 'Warning 1', description: 'Describes warning 1' },
+				{
+					name: 'Warning 2',
+					description: 'Describes warning 2',
+					supportUrl: 'https://helpme.com',
+				},
+			],
+		} );
+
+		const { getByText } = renderWithStore(
+			<EligibilityWarnings backUrl="" onProceed={ noop } />,
+			state
+		);
+
+		expect( getByText( 'Warning 1' ) ).toBeVisible();
+		expect( getByText( 'Describes warning 1' ) ).toBeVisible();
+		expect( getByText( 'Warning 2' ) ).toBeVisible();
+		expect( getByText( 'Describes warning 2' ) ).toBeVisible();
+
+		expect( getByText( 'Learn more.' ) ).toHaveAttribute( 'href', 'https://helpme.com' );
+	} );
+
+	it( "doesn't render warnings when there are blocking holds", () => {
+		const state = createState( {
+			holds: [ 'BLOCKED_ATOMIC_TRANSFER' ],
+			warnings: [
+				{
+					name: 'Warning',
+					description: 'Description',
+				},
+			],
+		} );
+
+		const { container } = renderWithStore(
+			<EligibilityWarnings backUrl="" onProceed={ noop } />,
+			state
+		);
+
+		expect( container.querySelectorAll( '.notice.is-warning' ) ).toHaveLength( 0 );
+	} );
+
+	it( 'goes to checkout when clicking "Upgrade and continue"', () => {
+		const state = createState( {
+			holds: [ 'NO_BUSINESS_PLAN', 'SITE_PRIVATE' ],
+			siteUrl: 'https://example.wordpress.com',
+		} );
+
+		const handleProceed = jest.fn();
+
+		const { getByText } = renderWithStore(
+			<EligibilityWarnings backUrl="" onProceed={ handleProceed } />,
+			state
+		);
+
+		const upgradeAndContinue = getByText( 'Upgrade and continue' );
+
+		fireEvent.click( upgradeAndContinue );
+
+		expect( handleProceed ).not.toHaveBeenCalled();
+		expect( upgradeAndContinue ).toHaveAttribute(
```

Comment:
This is the only test failing once the strings are fixed by #38562.

I think we need to do something like:
* Attach an `onClick` to the component which:
  * Dispatches the action to launch the site, then
  * Redirects to the checkout screen
* Update this test to cover the above

Reply:
That test is off, this isn’t supposed to work in this PR. For now the button is just greyed out. There is a separate PR to add „auto-launching” feature. It’s okay to remove this test from here.

Reply:
Ah I got confused by `Dispatches the action to launch the site, then` - I took a deeper look and this test should pass. Your follow-up PR will fix it though as the button will have the correct copy (https://github.com/Automattic/wp-calypso/pull/38562).